### PR TITLE
Feature/redesign 2

### PR DESCRIPTION
**Pull request by kirilknysh**
image

### CODE REVIEW

File: public/game/styles/game.styl [hunk 1]
```diff
@@ -21,46 +27,41 @@
...
+		color holy-white
 
-	.success-overlay
-		position absolute
-		top 0
-		left 0
-		width 100%
-		height 100%
-		background fog-60
+		&[disabled]
```
Comment (DaQuirm) [inline public/game/styles/game.styl]: How about `:disabled` instead?

File: public/game/styles/game.styl [hunk 2]
```diff
@@ -21,46 +27,41 @@
...
+			background lighter-than-background-disabled
+			color holy-white-disabled
+
+		&::-webkit-input-placeholder
```
Comment (DaQuirm) [inline public/game/styles/game.styl]: Those should be generated and added by Autoprefixer

File: public/common/stylesheets/occurrence-indicator.styl [hunk 3]
```diff
@@ -1,17 +1,31 @@
 @import './color-palette.styl'
 
+pattern-box-background = #426f6a
+pattern-box-matched-background = #745a57
+pattern-box-matched-border-stub = 2px solid transparent
+pattern-box-matched-border = 2px solid rgba(238, 84, 84, 0.45)
+
+//fix with complete font size integration
```
Comment (DaQuirm) [inline public/common/stylesheets/occurrence-indicator.styl]: An issue for that maybe?

File: public/game/views/game-view.coffee [hunk 4]
```diff
@@ -17,17 +17,34 @@ GameView = (context) ->
...
 						nxt.Element 'div',
 							nxt.Class 'controls-selector-banned-container'
-								nxt.Element 'div',
-									nxt.Class 'controls-selector-banned-container'
-									nxt.Text 'Banned characters'
-									OccurrenceIndicatorView context.occurrenceIndicator
+							nxt.Text 'Banned characters'
+							OccurrenceIndicatorView context.occurrenceIndicator
 
+				#TODO: move input to a separate component
```
Comment (DaQuirm) [inline public/game/views/game-view.coffee]: Maybe an issue for that too?..

File: public/common/stylesheets/occurrence-indicator.styl (hunk 3 above)
Comment (kirilknysh) [inline public/common/stylesheets/occurrence-indicator.styl]: I'll keep my eye on it, no worries

File: public/game/views/game-view.coffee (hunk 4 above)
Comment (kirilknysh) [inline public/game/views/game-view.coffee]: Will be fixed in next 1-2 PRs

**Comment by DaQuirm** [comment]
👍

**Pull request merged**